pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+# Overview:
+Give a brief overview of the issue you are solving. Succintly explain the GitHub issue you are addressin and the underlying problem of the ticket.
+
+# Addresses:
+Add a link to the issue on Github board
+
+# Type of Fix:
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Change
+
+# Solution:
+Outline your solution to the previously described issue and underlying cause. This section should include a brief description of your proposed solution and how it addresses the cause of the ticket.
+
+# Adds:
+Include a bulleted list or check box list of the implemented changes in brief, as well as the addition of supplementary materials(unit tests, integration tests, etc..)
+
+# Validation:
+Describe how you have validated that your solution addresses the root cause of the ticket. What have you done to ensure that your addition is bug free and works as expected. Please provide specific instructions so we can reproduce and list any relevant details about your configuration.
+- Screenshots
+- Unit Tests
+- Script to reproduce error
+- Configuration details
+
+# Checklist
+ - [ ] My code follows Google style guide
+ - [ ] My code is unit tested where appropriate and does not decrease test coverage
```

Comment:
"My code is unit-tested" -- all contributions should include unit tests.

Reply:
They should, so what's the harm in gently reminding people on the template.

Reply:
Sorry, I wasn't clear. I meant to truncate so that it just says, "My code is unit-tested." (removing the "where appropriate part, since it's always appropriate). If you want you can keep "does not decrease test coverage" but I think it's redundant.

Reply:
Oh I understand, my thought is that documentation updates and what not don't require unit tests. I'll keep the does not decrease test coverage as that will will cover both cases.